.github/workflows/ci.yml

```diff
@@ -320,11 +320,38 @@ jobs:
           profile: minimal
           override: true
 
-      - name: check docs in druid/
+      # Doc packages in deeper-to-higher dependency order
+      - name: cargo doc druid-shell
         uses: actions-rs/cargo@v1
         with:
           command: doc
-          args: --document-private-items
+          args: --manifest-path=druid-shell/Cargo.toml --document-private-items
+
+      - name: cargo doc druid
+        uses: actions-rs/cargo@v1
+        with:
+          command: doc
+          args: --manifest-path=druid/Cargo.toml --features=svg,image,im --document-private-items
+
+      - name: cargo doc druid-derive
+        uses: actions-rs/cargo@v1
+        with:
+          command: doc
+          args: --manifest-path=druid-derive/Cargo.toml --document-private-items
+
+      - name: cargo doc book examples
+        uses: actions-rs/cargo@v1
+        with:
+          command: doc
+          args: --manifest-path=docs/book_examples/Cargo.toml --document-private-items
```

Comment:
Seems consequent, but I'm not sure we need this one?

Reply:
Well might as well make sure that the doc format in the book examples is functional.